apps/code-examples/src/app/code-examples/forms/file-drop/help-key/demo.component.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import {
+  SkyHelpTestingController,
+  SkyHelpTestingModule,
+} from '@skyux/core/testing';
+
+import { DemoComponent } from './demo.component';
+
+describe('File drop', () => {
+  async function setupTest(): Promise<{
+    fixture: ComponentFixture<DemoComponent>;
+    helpController: SkyHelpTestingController;
+  }> {
+    const fixture = TestBed.createComponent(DemoComponent);
+
+    const helpController = TestBed.inject(SkyHelpTestingController);
+
+    fixture.detectChanges();
+
+    return { fixture, helpController };
+  }
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [DemoComponent, SkyHelpTestingModule],
+    });
+  });
+
+  it('should have the correct help key', async () => {
+    const { fixture, helpController } = await setupTest();
+
+    const helpButton = fixture.nativeElement.querySelector(
+      'sky-help-inline button',
```

Comment:
We should not add spec files for components which do not yet have a test harness.